ext/drcovlib/drcovlib.c

```diff
@@ -213,7 +213,7 @@ static void
 dump_drcov_data(void *drcontext, per_thread_t *data)
 {
     version_print(data->log);
-    module_table_print(module_table, data->log, false);
+    module_table_print(module_table, data->log, options.verbose_module_table);
```

Comment:
Looking it deeper, it looks like this would change the dumped data format, which makes the postprocessing not working, right?
In this case, maybe we should have a separate option to control.